.github/workflows/compile.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   build:
     needs: formatting-check
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 6
+    timeout-minutes: 15
```

Comment:
Please Revert this one

Reply:
You asked me to help fix your broken builds.  The build is timing out.  Increasing this value now has two of the builds passing.